crate/operator/main.py

```diff
@@ -105,6 +106,7 @@ async def cluster_create(
 
 
 @kopf.on.update(API_GROUP, "v1", RESOURCE_CRATEDB, id=CLUSTER_UPDATE_ID)
+@crate.on.error(error_handler=crate.send_error_notification)
```

Comment:
Would it be nicer to send a error notification for the specific operation that failed? i.e. instead of `WebhookEvent.ERROR` with `WebhookStatus.FAILURE` send a `WebHookEvent.UPGRADE` with `WebhookStatus.FAILURE`? 

Just thinking that it would be easier to map this to a running operations in the API then?..

```suggestion
@crate.on.error(error_handler=crate.send_upgrade_failed_notification)
```